packages/styling/src/styles/theme.ts

```diff
@@ -145,19 +145,22 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
 function _makeSemanticColorsFromPalette(p: IPalette, isInverted: boolean, depComments: boolean): ISemanticColors {
   let toReturn: ISemanticColors = {
     bodyBackground: p.white,
+    bodyStandoutBackground: p.white,
```

Comment:
you'll also want to add the new redlines in variants:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/variants/src/variants.ts
(might as well do it all at once)

Reply:
@phkuo what do you mean by "add the new redlines"?

Reply:
@phkuo Oh, do you mean add bodyStandoutBackground:neutralLight into getNeutralVariant() and getStrongVariant() inside the partialSemantic const? I can do that. Fingers crossed for no conflicts with the other PR about variants.ts.

Reply:
Ok, added. Let me know if there's anything further needed on that.